src/utils/classValidatorDecorators.ts

```diff
@@ -1,100 +1,224 @@
-export const classValidatorDecorators = [
-    "IsBooleanString",
-    "IsPositive",
-    "IsLatLong",
-    "IsLongitude",
-    "IsLatitude",
-    "IsNegative",
+export const typeCheckingDecorator = [
```

Comment:
i'm going to try to remove the need for these lists completely. They're too difficult to maintain